genai-perf/tests/test_llm_metrics.py

```diff
@@ -71,11 +71,13 @@ def test_llm_metric_system_metrics(self) -> None:
         )
 
         sys_metrics = m.system_metrics
-        assert len(sys_metrics) == 2
+        assert len(sys_metrics) == 3
         assert sys_metrics[0].name == "output_token_throughput"
         assert sys_metrics[0].unit == "per sec"
-        assert sys_metrics[1].name == "request_throughput"
+        assert sys_metrics[1].name == "request_goodput"
```

Comment:
You want to write tests specifically for goodput. My recommendation on the unit testing side is at least these three:
1. What happens when 1 goodput constraint is added?
2. What happens when 3 goodput constraints are added?
3. What happens when 2 goodput constraints are added and one does not exist in the model?

Reply:
Thanks for the comment. This is not a start for unit testing work.
Unit tests work for goodput is another todo ticket.
This one is just one of previous unit tests, if I don't make these changes, it does not pass locally.

Reply:
Updated